src/main/java/Duke.java

```diff
@@ -1,10 +1,40 @@
+import Duke.Tool.*;
+import Duke.Task.*;
```

Comment:
Perhaps it may be better to list the imported classes explicitly so that others can easily identify what classes are yoused?